test/.clang-format

```diff
@@ -0,0 +1,6 @@
+---
+Language:      Cpp
+BasedOnStyle:  LLVM
+IndentWidth:   4
+InsertBraces: true
+...
```

Comment:
Please use `ReflowComments: false`. See #267 for context.
Otherwise lgtm.

Reply:
I think the fact that it is non-idempotent won't affect code that is not autogenerated.

Reply:
It might, because in CI we are essentially formatting the entire repo and failing if there's a diff. So it doesn't matter whether it's autogenerated or not.

Reply:
👍